website/content/en/docs/olm-integration/testing-deployment.md

```diff
@@ -8,51 +8,141 @@ This document discusses the behavior of `operator-sdk <run|cleanup>` subcommands
 and assumes you are familiar with [OLM][olm], related terminology,
 and have read the SDK-OLM integration [design proposal][sdk-olm-design].
 
-Currently only the package manifests format is supported by `<run|cleanup>` subcommands. Bundle support is coming soon.
-
 **Note:** before continuing, please read the [caveats](#caveats) section below.
 
-## `operator-sdk <run|cleanup> packagemanifests` command overview
+## `operator-sdk run bundle` command overview
```

Comment:
This doc should be redone with examples instead of discussing the CLI, since the flag help should be sufficient in most cases and this doc can drift as flags are added/help is changed. Nothing to do right now though.